README.md

```diff
@@ -105,7 +121,21 @@ Before starting any work, please either comment on an existing issue, or file a
 
 ## Supportability
 
-Currently, Kubebuilder officially supports OSX and Linux platforms. So, if you are using a Windows OS you may find issues. Contributions towards supporting Windows are welcome.
+Currently, Kubebuilder officially supports OSX and Linux platforms. 
+So, if you are using a Windows OS you may find issues. Contributions towards 
+supporting Windows are welcome.
+
+### Apple Silicon
+
+The current scaffold done by the CLI (`go/v3`) when we run `kubebuilder init` uses [kubernetes-sigs/kustomize][kustomize] v3 which does not provide
+a valid binary for Apple Silicon (`darwin/arm64`). Therefore, you can use the `go/v4-alpha` plugin
+instead which provides support for this platform:
```

Comment:
You could simplify this and just let them know that M1 is supported using v4.

```suggestion
Apple Silicon (`darwin/arm64`) is supported using the `go/v4-alpha` plugin which provides support for this platform.
```

If you must let them know why it is broken, then link to an issue. But having the full explanation as to why it doesn't work just makes it wordy.

```suggestion
Apple Silicon (`darwin/arm64`) is supported using the `go/v4-alpha` plugin which provides support for this platform. There is an [issue](https://github.com/kubernetes-sigs/kubebuilder/issues/2566) with the current `go/v3` plugin and Apple Silicon.
```

Reply:
@jmrodri it is a good point. Changing